2025-02-12 09:29:55,501 INFO root MainThread : /tendermint/node0/config/config.toml
Starting Tendermint: ['tendermint', 'node', '--proxy_app=tcp://newserviceufff_abci_0:26658', '--rpc.laddr=tcp://0.0.0.0:26657', '--p2p.laddr=tcp://0.0.0.0:26656', '--p2p.seeds=', '--consensus.create_empty_blocks=true', '--abci=socket', '--home', '/tendermint/node0']
Tendermint process started
2025-02-12 09:29:55,510 WARNING werkzeug MainThread :  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2025-02-12 09:29:55,510 INFO werkzeug MainThread :  * Running on http://192.167.13.3:8080/ (Press CTRL+C to quit)
Monitoring thread started
I[2025-02-12|09:29:55.611] Starting multiAppConn service                module=proxy impl=multiAppConn
I[2025-02-12|09:29:55.612] Starting socketClient service                module=abci-client connection=query impl=socketClient
E[2025-02-12|09:29:55.614] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp 192.167.13.2:26658: connect: connection refused"
E[2025-02-12|09:29:58.643] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:01.668] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:04.684] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:07.701] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:10.714] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:13.735] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:16.767] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:20.046] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:23.058] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:26.077] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:29.089] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:32.100] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:35.117] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:38.130] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:41.142] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:44.160] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:47.173] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:50.188] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:53.202] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:56.218] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:30:59.248] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:02.270] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:05.294] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:08.314] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:11.330] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:14.347] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:17.368] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:20.389] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:23.407] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:26.426] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:29.445] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:32.460] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:35.485] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:38.501] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:41.514] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:44.529] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:47.560] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:50.569] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:53.582] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:56.597] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:31:59.621] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:02.644] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:05.663] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:08.679] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:11.691] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:14.721] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:17.730] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:20.747] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:23.759] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:26.776] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:29.835] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:32.866] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:35.882] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:38.896] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:41.921] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:44.941] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:47.966] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:50.996] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:54.009] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:32:57.026] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:00.042] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:03.052] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:06.063] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:09.086] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:12.093] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:15.103] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:18.115] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:21.134] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:24.148] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:27.170] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:30.189] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:33.203] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:36.238] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:39.254] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:42.273] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:45.289] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:48.303] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:51.317] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:54.332] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:33:57.348] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:00.359] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:03.371] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:06.380] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:09.401] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:12.415] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:15.435] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:18.450] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:21.474] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:24.492] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:27.526] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:30.540] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:33.562] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:36.573] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:39.586] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:42.604] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:45.626] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:48.639] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:51.654] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:54.665] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:34:57.676] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:00.694] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:03.711] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:06.717] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:09.733] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:12.755] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:15.772] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:18.785] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:21.814] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:25.009] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:28.020] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:31.026] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:34.045] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:37.055] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:40.068] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:43.078] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:46.089] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
E[2025-02-12|09:35:49.109] abci.socketClient failed to connect to tcp://newserviceufff_abci_0:26658.  Retrying after 3s... module=abci-client connection=query err="dial tcp: lookup newserviceufff_abci_0 on 127.0.0.11:53: no such host"
